pkgs/by-name/sp/spiel/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  stdenv,
+  python312Packages,
+  callPackage,
```

Comment:
```suggestion
```
The `by-name` overlay automatically calls for your package. No need to use it anywhere in your package.